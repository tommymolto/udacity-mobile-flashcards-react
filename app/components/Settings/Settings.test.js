/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import Settings from './Settings';

describe('Settings', () => {
  it('renders correctly', () => {
    const navigation = {
      navigate: 'Notifications',
    };
    const tree = renderer.create(<Settings navigation={navigation} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
