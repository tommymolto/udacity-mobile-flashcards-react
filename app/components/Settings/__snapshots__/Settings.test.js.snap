// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 24,
          "paddingHorizontal": 15,
          "paddingVertical": 15,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Settings screen
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "#BDC7CF",
            "color": "#000000",
            "fontSize": 22,
            "marginLeft": 5,
            "marginRight": 5,
            "marginVertical": 10,
            "paddingBottom": 10,
            "paddingTop": 10,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      Notifications
    </Text>
  </View>
</View>
`;
